packages/suite/src/hooks/wallet/useSendFormOutputs.ts

```diff
@@ -86,27 +81,21 @@ export const useSendFormOutputs = ({
                         },
                     ],
                 },
-                { errors: true },
+                { keepErrors: true },
             );
         }
-        composeRequest('outputs[0].amount');
+        composeRequest('outputs.0.amount');
     };
 
-    // each Output have additional uncontrolled values that needs to be present in FormState
+    // each Output has additional uncontrolled values that need to be present in FormState
     // they need to be registered without any HTMLElement as a "custom" field
     const { fields } = outputsFieldArray;
     useEffect(() => {
         fields.forEach((output, index) => {
-            register({ name: `outputs[${index}].type`, type: 'custom' });
+            register(`outputs.${index}.type`, { shouldUnregister: true });
             // set defaultValues
-            setValue(`outputs[${index}].type`, output.type);
+            setValue(`outputs.${index}.type`, output.type);
         });
-        return () => {
-            // unregister fields
-            fields.forEach((_output, index) => {
-                unregister(`outputs[${index}].type`);
-            });
-        };
     }, [fields, register, unregister, setValue]);
```

Comment:
`unregister` is not required anymore. Weird that linter does not yell. 
It can also be deleted from props

Reply:
c358bc0407a67c76e6e1a981e1ed81bb9383f099